Portage/GHCCore.hs

```diff
@@ -144,6 +148,40 @@ ghc741 = (ghc [7,4,1], mkIndex ghc741_pkgs)
 -- Source: http://haskell.org/haskellwiki/Libraries_released_with_GHC
 --         and our binary tarballs (package.conf.d.initial subdir)
 
+
+ghc821_pkgs :: [Cabal.PackageIdentifier]
+ghc821_pkgs =
+  [ p "array" [0,5,2,0]
+  , p "base" [4,10,0,0]
+  , p "binary" [0,8,5,1] -- used by libghc
+  , p "bytestring" [0,10,8,2]
+--  , p "Cabal" [2,0,0,0]  package is upgradeable
+  , p "containers" [0,5,10,2]
+  , p "deepseq" [1,4,3,0] -- used by time
+  , p "directory" [1,3,0,2]
+  , p "filepath" [1,4,1,2]
+  , p "ghc-boot" [8,2,1]
+--  , p "ghc-boot-th" [] err... vOv
```

Comment:
I think it should be added visible. It's not an upgradeable, but visible package:
`ghci -package=ghc-boot-th` works fine.